rpc_jobs/standard_job_premerge.yml

```diff
@@ -1,7 +1,7 @@
 - project:
-    name: "gating"
+    name: "gating-pre-merge"
     repo_url: "https://github.com/rcbops/rpc-gating"
-    branches:
+    branch:
```

Comment:
See the docs - this is a list of branch names to trigger from which is evaluated by jenkins as a list of regex entries.

Reply:
As discussed in person I disagree, but will go with branches.